### PR TITLE
Create requirements.txt

### LINKED ISSUE #1: how to resolve depency issues?
Sorry for the noob question.
But is there a requirements.txt somewhere?

Running the setup.py is not working for me.
How do I install all the required packages for an M1 mac?

### PR DESCRIPTION
**Pull request by Naman23-coder**
closes #1

### CODE REVIEW

**Comment by ltrottier** [comment]
It is missing the `tensorflow` package

**Comment by SumitBando** [comment]
This is missing tensorflow, regex and Pillow.

For my 2019 Intel based Macbook, to be able to run the documented example: python text2image.py --prompt="An astronaut riding a horse", working requirements.txt:

absl-py==1.2.0
astunparse==1.6.3
cachetools==5.2.0
certifi==2022.9.14
charset-normalizer==2.1.1
colorama==0.4.5
flatbuffers==2.0.7
ftfy==6.1.1
gast==0.4.0
google-auth==2.11.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
grpcio==1.49.0
h5py==3.7.0
idna==3.4
importlib-metadata==4.12.0
keras==2.7.0
Keras-Preprocessing==1.1.2
libclang==14.0.6
Markdown==3.4.1
MarkupSafe==2.1.1
numpy==1.23.3
oauthlib==3.2.1
opt-einsum==3.3.0
packaging==21.3
Pillow==9.2.0
protobuf==3.19.5
pyasn1==0.4.8
pyasn1-modules==0.2.8
pyparsing==3.0.9
regex==2022.9.13
requests==2.28.1
requests-oauthlib==1.3.1
rsa==4.9
six==1.16.0
tensorboard==2.10.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.7.4
tensorflow-addons==0.15.0
tensorflow-estimator==2.7.0
tensorflow-io-gcs-filesystem==0.27.0
termcolor==2.0.1
tqdm==4.64.1
typeguard==2.13.3
typing_extensions==4.3.0
urllib3==1.26.12
wcwidth==0.2.5
Werkzeug==2.2.2
wrapt==1.14.1
zipp==3.8.1

**Comment by Naman23-coder** [comment]
@ltrottier might be pip freeze error

**Comment by Naman23-coder** [comment]
Will add the missing packages
